providers/flagd/pkg/provider_test.go

```diff
@@ -116,13 +116,14 @@ func TestNewProvider(t *testing.T) {
 		}
 		svc := flagd.NewProvider(test.options...)
 		if svc == nil {
-			t.Error("received nil service from NewProvider")
+			t.Errorf("%s received nil service from NewProvider", test.name)
```

Comment:
`t.Fatalf(`
removes the need for `t.FailNow()`

Reply:
resolved, and removed other examples of this